Cargo.toml

```diff
@@ -12,14 +12,19 @@ csv = "1.1"
 dotenv = "^0.15"
 env_logger = "^0.7"
 failure = "*"
-futures = "0.1"
-gm = { git = "https://github.com/paritytech/glitch-in-the-matrix", package="glitch-in-the-matrix" }
+futures = "0.3"
 hyperx = "0.15.2"
 log = "^0.4"
+rayon = "1.2"
 reqwest = "^0.9"
 rocksdb = "0.13"
 serde = "^1.0"
 serde_json = "1.0.44"
 snafu = { version = "0.6.0", features = ["backtraces"] }
 tokio = { version = "0.2", features = ["full"] }
-tokio-core = "0.1"
+curl = "0.4"
+curl-sys = "0.4"
+regex = "1.3"
+toml = "0.5"
+anyhow = "1.0"
```

Comment:
We don't have to block this PR on it, but we should decide on depending on a single library for error handling.